Examples/Tests/SingleParticle/inputs

```diff
@@ -1,23 +1,137 @@
-max_step = 1
```

Comment:
This file is used in another test (bilinear_filter). Modifying it broke the other test.

Reply:
Sorry, modifying this file was a mistake. I've just reverted it to its previous version